MeetU/MeetU/Views/Meetups/Index.cshtml

```diff
@@ -10,73 +10,138 @@
         <div class="col-md-1"></div>
         <div class="col-md-8 ng-cloak">
             <div ng-repeat="meetupView in meetupViews">
-                <div class="panel panel-default">
-                    <div class="panel-heading">
-                        <div class="btn-group btn-group-xs" role="group">
-                            <button type="button" class="btn btn-info" ng-click="toggleJoin(meetupView)">
-                                <span> {{isIn(meetupView.joins) ? 'Quit' : 'Join'}} </span>
-                            </button>
-                            <button type="button" class="btn btn-success">
-                                <span> {{ meetupView.joins.length }} </span>
-                            </button>
+                <div class="meetup-wrapper">
+                    <div class="meetup-brief" ng-click="showDetails = ! showDetails">
+                        <div class="meetup-from col-sm-11">
+                            <span>@@{{ meetupView.sponsorUserName }} 发起了活动</span>
```

Comment:
@rong4188 
Can we use English here? Something like `@someone post a new meetup` 
At least use English before my job search is done..